content/client-server-api/modules/push.md

```diff
@@ -382,6 +382,9 @@ The following `alt_aliases` values will NOT match:
 
 **`contains_display_name`**
 
+{{% changed-in v="1.16" %}}: this rule is deprecated and **should not be used in
```

Comment:
```suggestion
{{% changed-in v="1.16" %}}: this condition is deprecated and **should not be used in
```

Reply:
Applied in https://github.com/matrix-org/matrix-spec/pull/2186/commits/7b5935e5cb430c45804b82147813488aee2b869d.